src/pages/docs/guides/chat/replies.mdx

```diff
@@ -0,0 +1,803 @@
+---
+title: "Implementing Replies and Quotes with Chat"
+meta_description: "Learn how to implement message replies and quotes in your Ably Chat applications using TypeScript. This guide shows how to use metadata to create quote and reply functionality."
+meta_keywords: "ably chat, message replies, message quotes, chat replies, chat quotes, javascript chat replies, typescript chat replies, chat metadata, ably chat metadata, realtime chat replies"
+---
+
+This guide will show you how you could implement a simple solution for things like replies and quotes in your Ably Chat application.
+While the two features have different applications, they share the same fundamental principle of referencing other messages in the chat.
+As such, we will cover only replies in this guide, but the same principles can be applied to quotes.
+
+While cross-room replies are possible, this guide will focus on providing replies within the same room for simplicity.
+
+## Understanding the current limitations <a id="limitations"/>
+
+While Ably Chat doesn't have built-in support for replies or quotes, you can implement this functionality by using the metadata fields that can be sent along with chat messages.
+
+Every message in Ably Chat has:
+- A `serial` field (a UUID that uniquely identifies the message)
+- A `createdAt` field (when the message was created)
+- A `metadata` field (key-value store for custom data)
+
+By storing the serial and creation time of a message you want to quote in the metadata of a new message, you can create a reference between messages that your application can interpret and display accordingly.
+
+## Implementing replies using message metadata <a id="implementing-replies"/>
+
+To implement a reply to a message, you'll need to:
+
+1. Store the original message's information when a user wants to reply to it.
+2. Include that information in the metadata when sending the reply.
+3. Process incoming messages to check for reply metadata.
+
+### Step 1: Store information about the message being replied to <a id="store-information"/>
```

Comment:
I think we should move away from making this step-based and just discuss the process as we normally would for another feature.